shapely/_geometry.py

```diff
@@ -499,24 +522,25 @@ def get_interior_ring(geometry, index, **kwargs):
     True
     >>> get_interior_ring(Point(0, 0), 0) is None
     True
+
     """
     return lib.get_interior_ring(geometry, np.intc(index), **kwargs)
 
 
 @multithreading_enabled
 def get_num_interior_rings(geometry, **kwargs):
-    """Returns number of internal rings in a polygon
+    """Return number of internal rings in a polygon.
 
     Returns 0 for not-a-geometry values.
 
     Parameters
     ----------
     geometry : Geometry or array_like
-        The number of interior rings in non-polygons equals zero.
```

Comment:
This information is lost now? (there is an example that shows it, but I think would be good to keep explicitly in the text as well)

Reply:
A bit below, in `get_num_geometries`, you moved the content like this to the expanded description above (append the "Returns 0 for not-a-geometry values." sentence)